ras_party/controllers/respondent_controller.py

```diff
@@ -40,3 +41,17 @@ def get_respondent_by_email(email, session):
         raise RasError("Respondent does not exist.", status=404)
 
     return respondent.to_respondent_dict()
+
+
+@with_db_session
+def change_respondent_details(respondent_data, session):
+    """
+    :param respondent_data:
+    :param session:
+    :return:
+    """
+    v = Validator(IsUuid('respondent_id'))
+    if not v.validate({'respondent_id': respondent_data['respondent_id']}):
+        raise RasError(v.errors, status=404)
```

Comment:
I think this should be a 400 if the id is invalid

Reply:
Yeah agreed. This is a check to see if the format of the `respondent_id` is correct, not to check if it exists